integration/update_field_compat_test.go

```diff
@@ -212,17 +212,17 @@ func TestUpdateFieldCompatIncComplex(t *testing.T) {
 		"DotNotationNegativeIndex": {
 			update: bson.D{{"$inc", bson.D{{"v.-1", int32(42)}}}},
 		},
-		"DotNotationIndexOutsideArray": {
+		"DotNotIndexOutsideArray": {
 			update: bson.D{{"$inc", bson.D{{"v.100", int32(42)}}}},
 		},
-		"DotNotationArrayFieldNotExist": {
+		"DotNotArrayFieldNotExist": {
```

Comment:
"Not array field not exists". While the test is actually about array, not "not array".

I think that abbreviation of dot notation is unfortunate. It also does not quite belong to that PR/issue.

Reply:
We need to shorten those names as they are longer than 64 characters so they always fail

Reply:
I'm not sure why they become longer if we don't change them, but okay, let's shorten them some other non-confusing way

Reply:
They are already too long since creation. We just don't see it as they fail in the same way both on MongoDB and FerretDB. We could actually panic or fail if the name is too long to omit such situation in the future

Reply:
Oh, that's bad…

Reply:
@noisersup is it done?

Reply:
I replaced the names with less confusing ones, currently waiting for CI to pass, I'm also investigating why the tests were passing, although not sure if that should be covered in this PR in your opinion

Reply:
I think we should at least understand why that was happening

Reply:
Found it, it's due to the fact that we append `_sqlite` at the end of the database name

Reply:
If we want to avoid name mangling I guess we are forced to shorten our db names.